influxdb/src/main/scala/akka/stream/alpakka/influxdb/InfluxDBWriteMessage.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.influxdb
+
+import akka.NotUsed
+
+object InfluxDBWriteMessage {
+  // Apply method to use when not using passThrough
+  def apply[T](point: T): InfluxDBWriteMessage[T, NotUsed] =
+    InfluxDBWriteMessage(point = point, passThrough = NotUsed)
+
+  // Java-api - without passThrough
+  def create[T](point: T): InfluxDBWriteMessage[T, NotUsed] =
+    InfluxDBWriteMessage(point, NotUsed)
+
+  // Java-api - with passThrough
+  def create[T, C](point: T, passThrough: C) =
+    InfluxDBWriteMessage(point, passThrough)
+}
+
+final case class InfluxDBWriteMessage[T, C](point: T,
```

Comment:
Make this just a regular class with a private constructor, please.

Reply:
I was using it for pattern matching. Will proceed on the change as advised.